azure/Guidelines.md

```diff
@@ -374,6 +375,10 @@ Services, and the clients that access them, may be written in multiple languages
 
 :white_check_mark: **DO** use [RFC3339](https://datatracker.ietf.org/doc/html/rfc3339) for date/time.
 
+:white_check_mark: **DO** use a fixed time interval when using durations and include the time unit in the name e.g., `backupTimeInMinutes` or `ttlSeconds`. The time unit should also be included in the property description to support localization e.g., "How often to back up expressed in minutes," or "The lifetime of the resource expressed in seconds."
```

Comment:
```suggestion
:white_check_mark: **DO** use a fixed time interval to express durations and include the time unit in the property name e.g., `backupTimeInMinutes` or `ttlSeconds`. The time unit should also be included in the property description to support localization e.g., "How often to back up expressed in minutes," or "The lifetime of the resource expressed in seconds."
```

Reply:
I don't know how this is done (automated or manual) and if done manually, I don't know if this happens for all services. Regardless, we never mention anything about localization in any of our guidelines and so I'd prefer we not mention it here as mentioning it introduced questions and uncertainty.